base/set.jl

```diff
@@ -1,5 +1,33 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+	Set{T} <: AbstractSet{T}
+
+The default implementation of hash sets.
+
+`Set`s are containers that provide fast lookup and membership testing.
+Elements in a `Set` are unique, as determined by the elements' implementation of
+`isequal` and `hash`.
+The order of elements in a `Set` is an implementation detail and cannot be relied on.
+
+See also: [`AbstractSet`](@ref), [`BitSet`](@ref), [`Dict`](@ref)
+
+# Examples
```

Comment:
Re examples, it would be nice to show `in`, and to show some example that would fail if the check were `==` or `===` not `isequal`. Maybe:
```
julia> 0.0 in Set([-0.0, NaN, Inf])  # 0.0 == -0.0 is true, but Set uses isequal
false

julia> [2] in Set(([1], [2], [3]))  # [2] === [2] is false, but Set uses isequal
true
```

Reply:
In examples, added `0` vs `0.0` vs `-0.0`.